common/workunit/workunit.cpp

```diff
@@ -13932,6 +13932,12 @@ void waitK8sJob(const char *componentName, const char *job)
         {
             // Job is no longer active - we can terminate
             DBGLOG("kubectl jobs output: %s", output.str());
+            VStringBuffer checkJob("kubectl get pods --selector=job-name=%s --output=jsonpath={.items[*].status.containerStatuses[?(@.name==\"%s\")].state.terminated.exitCode}", jobname.str(), jobname.str());
```

Comment:
minor: for consistency should move this out of the loop like the other VStringBuffers